pinot-integration-tests/src/test/resources/On_Time_On_Time_Performance_2014_100k_subset_nonulls_ingestion_config.schema

```diff
@@ -0,0 +1,342 @@
+{
```

Comment:
schema contains some fields which are not in the source data. I preferred adding a new file instead of putting it directly in the test file. But changed it now to override createSchema